plugin/eleline.vim

```diff
@@ -220,6 +235,10 @@ endfunction
 
 " https://github.com/liuchengxu/eleline.vim/wiki
 function! s:StatusLine() abort
+  if s:is_tmp_file()
+    return ''
+  endif
```

Comment:
Why this?

Reply:
It could be removed.  But a bit annoying and distracting to show the status bar for vista, defx, luatree etc.

Reply:
Let's remove it and focus on the lsp support in this PR.

Reply:
Sure, done.